### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by JohnTitor**
Signed-off-by: Yuki Okushi <jtitor@2k36.org>

### CODE REVIEW

File: src/opaque-types-impl-trait-inference.md [hunk 1]
```diff
@@ -216,15 +216,15 @@ and then handle it correctly.
...
+but ignores obviously dead code (e.g. after a panic).
 The borrow checker is also the source of truth when it comes to hidden types,
 as it is the only one who can properly figure out what lifetimes on the hidden type correspond
 to which lifetimes on the opaque type declaration.
 
 ## Backwards compatibility hacks
 
 `impl Trait` in return position has various quirks that were not part
-of any RFCs and are likely accidental stabilizations.
+of any RFCs and are likely accidental stabilization.
```
Comment (tshepang) [inline src/opaque-types-impl-trait-inference.md:227]:
```suggestion
of any RFCs and are likely an accidental stabilization.
```
is this what was meant

Comment (JohnTitor) [reply src/opaque-types-impl-trait-inference.md:227]: I'm not sure but does it become a "countable noun" in this context?

Comment (tshepang) [reply src/opaque-types-impl-trait-inference.md:227]: hm, I think it would instead have to be `... and is likely an accidental stabilization.`

Comment (JohnTitor) [reply src/opaque-types-impl-trait-inference.md:227]: Well, I'm talking about if it's a countable noun or an uncountable one and I think it depends on the context, but I'm not sure if this is the case.

Comment (tshepang) [reply src/opaque-types-impl-trait-inference.md:227]: I don't know... it just didn't read well, or was not obvious, hence the suggested edit.

**Pull request merged**